specification/communication/data-plane/CallingServer/preview/2022-04-07-preview/communicationservicescallingserver.json

```diff
@@ -0,0 +1,2101 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-04-07-preview",
+    "title": "Azure Communication Service ServerCalling APIs",
+    "description": "Azure Communication Service ServerCalling APIs"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/calling/callConnections": {
+      "post": {
+        "tags": [
+          "PreConnections"
+        ],
+        "summary": "Create an outbound call.",
+        "description": "Create an outbound call.",
+        "operationId": "ServerCalling_CreateCall",
+        "parameters": [
+          {
+            "name": "createCallRequest",
+            "in": "body",
+            "description": "The create call request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateCallRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "Repeatability-Request-ID",
+            "in": "header",
+            "description": "If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-Id and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-Id is an opaque string representing a client-generated unique identifier for the request. It is a version 4 (random) UUID.",
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "Repeatability-First-Sent",
+            "in": "header",
+            "description": "If Repeatability-Request-ID header is specified, then Repeatability-First-Sent header must also be specified. The value should be the date and time at which the request was first created, expressed using the IMF-fixdate form of HTTP-date. Example: Sun, 06 Nov 1994 08:49:37 GMT.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Returns the create call response.",
+            "schema": {
+              "$ref": "#/definitions/CallConnectionProperties"
+            }
+          }
```

Comment:
All operations should have a "default" response for errors.

Reply:
Added